contrib/kubespray/quick-start-kubespray.sh

```diff
@@ -1,5 +1,22 @@
 #!/bin/bash
 
+if getopts "v" opt; then
+  case $opt in
+    v)
+      export ANSIBLE_DISPLAY_OK_HOSTS=yes
+      export DISPLAY_SKIPPED_HOSTS=yes
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG"
+      exit 1
+      ;;
+  esac
+else
+    export ANSIBLE_DISPLAY_OK_HOSTS=no
+    export DISPLAY_SKIPPED_HOSTS=no
+    export ANSIBLE_CALLBACK_WHITELIST=""
```

Comment:
what should `ANSIBLE_CALLBACK_WHITELIST` be in `-v` mode ?

Reply:
We didn't specified it. In `-v` mode, it can be customized by the user.